.gitignore

```diff
@@ -76,6 +76,8 @@ atlassian-ide-plugin.xml
 # Cursive Clojure plugin
 .idea/replstate.xml
 
+.idea/
```

Comment:
https://intellij-support.jetbrains.com/hc/en-us/articles/206544839

Mentions there might be files in .idea we want to include.  Should we have the whole folder ignored?